core/src/main/resources/hudson/model/UpdateCenter/sidepanel.jelly

```diff
@@ -30,8 +30,6 @@ THE SOFTWARE.
   <l:header />
   <l:side-panel>
     <l:tasks>
-      <l:task href="${rootURL}/" icon="icon-up icon-md" title="${%Back to Dashboard}"/>
-      <l:task href="${rootURL}/manage" icon="symbol-settings" title="${%Manage Jenkins}"/>
       <l:task href="../pluginManager/" icon="icon-plugin icon-lg" title="${%Manage Plugins}"/>
```

Comment:
For pages that retain a sidepanel with links elsewhere, would it make sense to have a "You are here" item, like "Status" on the job overview page?

Reply:
Good idea 👍 I'll look into that

Reply:
Done 👍 Can be updated further once #6783 is in

<img width="580" alt="image" src="https://user-images.githubusercontent.com/43062514/180661865-c94b63b7-93f5-46db-aaa1-84af035b83db.png">